backend/dataall/modules/shares_base/db/share_object_state_machines.py

```diff
@@ -264,23 +264,6 @@ def update_state(self, session, share_uri, new_state):
 
     def update_state_single_item(self, session, share_item, new_state):
         logger.info(f'Updating share item in DB {share_item.shareItemUri} status to {new_state}')
-        ShareObjectRepository.update_share_item_status(session=session, uri=share_item.shareItemUri, status=new_state)
+        ShareStatusRepository.update_share_item_status(session=session, uri=share_item.shareItemUri, status=new_state)
         self._state = new_state
         return True
-
```

Comment:
Moved get_share_item_shared_states and get_...  moved to ShareStatusRepository to avoid circular dependencies